std/algorithm/comparison.d

```diff
@@ -1382,6 +1385,8 @@ Iterates the passed arguments and returns the minimum value.
 Params: args = The values to select the minimum from. At least two arguments
     must be passed, and they must be comparable with `<`.
 Returns: The minimum of the passed-in values.
+See_Also:
+    $(XREF algorithm,searching,minElement)
```

Comment:
I've tried to migrate all our cross-referencing over to the new variadic `MREF` and `REF` macros, but the PRs are stagnant. Without these new macros our situation is a mess. I'd be up for documenting our Phobos macros once we pull through with that.